admin_manual/configuration_server/occ_command.rst

```diff
@@ -1650,24 +1650,21 @@ or to use in a bug report::
 
 Two-factor authentication
 -------------------------
-If a two-factor provider app is enabled, it is enabled for all users by default
-(though the provider can decide whether or not the user has to pass the challenge).
-In the case of a user losing access to the second factor (e.g. lost phone with
-two-factor SMS verification), the admin can try to disable the two-factor
-check for that user via the occ command::
+If a two-factor provider app is enabled, it is enabled for all users by default (though the provider can decide whether or not the user has to pass the challenge). In the case of a user losing access to the second factor (e.g. lost phone with two-factor SMS verification), the admin can try to disable the two-factor check for that user via the occ command:
```

Comment:
The formatting of this paragraph was ok, wasn't it? (https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#two-factor-authentication), also code style about line length (max. 80 characters).